engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -264,6 +308,18 @@ public void update(float delta) {
         }
     }
 
+    /**
+     Called whenever there's a change to WorldConfigurator properties
+     Provides information about the changed property as well as old and new value in {@link PropertyChangeEvent}
+     */
```

Comment:
```suggestion
    /**
      * Called whenever there's a change to WorldConfigurator properties.
      * Provides information about the changed property as well as old and new value in {@link PropertyChangeEvent}.
      */
```

I am not sure if you need the prepended asterisks to work as valid JavaDoc but that's the style used everywhere else.

Reply:
done